src/Logger.ts

```diff
@@ -0,0 +1,45 @@
+import { createLogger as winstonCreateLogger, format, transports } from 'winston'
+const { colorize, combine, printf, json, timestamp } = format
+
+const consoleFormat = printf(({ level, message, timestamp, scriptId }) => {
+  return `${timestamp} ${scriptId ? 'script' : 'runtime'} ${level} ${message}`
+})
+
+const filterLog = (withScriptId: boolean) =>
+  format((info) => (info.scriptId ? (withScriptId ? info : false) : withScriptId ? false : info))
```

Comment:
Can do something like `(typeof info.scriptId === 'undefined') == withScriptId ? info : false`

Reply:
I'll remove this function, it will be in the enterprise but surely we can simplify the logic